vendor/sigs.k8s.io/kustomize/kyaml/internal/forked/github.com/go-yaml/yaml/LICENSE

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
+ @dims for nested forked licenses not getting included correctly, xref https://github.com/kubernetes/kubernetes/issues/103551

Reply:
in this case, we happen to already include LICENSES/vendor/gopkg.in/yaml.v3 and LICENSES/vendor/gopkg.in/yaml.v2